creating-virtual-machines/interfaces-and-networks.adoc

```diff
@@ -283,6 +289,12 @@ through a linux ``bridge''. The pod network IPv4 address is delegated to
 the virtual machine via DHCPv4. The virtual machine should be configured
 to use DHCP to acquire IPv4 addresses.
 
+________________________________________________________________________________________________________________________________________________________________________________________________________________________________
+**Note:** If a specific MAC address is not configured in the virtual machine interface spec
+the MAC address from the relevant pod interface is delegated to the virtual machine.
+________________________________________________________________________________________________________________________________________________________________________________________________________________________________
```

Comment:
text seems fine. I find it hard to see the relevance of the former version of the text, but never mind that. bygone.